Gemfile

```diff
@@ -18,6 +18,7 @@ gem "interactor"
 gem "kaminari"
 gem "rack-cors", require: "rack/cors"
 gem "responders"
+gem "rollbar"
```

Comment:
We may have explicit use `Rollbar` in the app, e.g. `Rollbar.debug("some info", user_id: current_user.id)` and in this case we should have `Rollbar` for each env (like in the [rails-base](https://github.com/fs/rails-base/blob/master/Gemfile#L41) template)